plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/startup/AmazonQStartupActivity.kt

```diff
@@ -30,6 +35,12 @@ class AmazonQStartupActivity : ProjectActivity {
     override suspend fun execute(project: Project) {
         if (ApplicationManager.getApplication().isUnitTestMode) return
 
+        ToolkitConnectionManager.getInstance(project).activeConnectionForFeature(QConnection.getInstance())?.let {
+            if (it is AwsBearerTokenConnection && isInternalUser(it.startUrl)) {
+                CodeWhispererSettings.getInstance().toggleProjectContextEnabled(true, passive = true)
```

Comment:
Can we also add the parameter name before the first boolean?

Reply:
okk let me update it

Reply:
added